packages/content-management/table_list_view_table/src/reducer.tsx

```diff
@@ -63,7 +64,10 @@ export function getReducer<T extends UserContentCommonSchema>() {
         };
       }
       case 'onSearchQueryChange': {
-        if (action.data.text === state.searchQuery.text) {
+        const clause = state.searchQuery.query.ast.clauses;
+        const newClause = action.data.query.ast.clauses;
```

Comment:
Without checking the change between clauses leads to querySearch not updated:


https://github.com/elastic/kibana/assets/6295984/0a81ab10-d7e5-4629-bfff-d7bde4efe7c2

Reply:
On the page load, we can see `tag("Security Solution")` in the search box, but non of the tag filter is selected. Expect that both Security Solution should be selected and shows the correct active count.